tests/functions/test_voyageai.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+import pixeltable as pxt
+
+from ..utils import rerun, skip_test_if_no_client, skip_test_if_not_installed, validate_update_status
+
+
+@pytest.mark.remote_api
+@rerun(reruns=3, reruns_delay=8)
+class TestVoyageAI:
+    def test_embeddings(self, reset_db: None) -> None:
```

Comment:
This is missing a test to use the embeddings in an index (which would fail right now).

Use the document splitter with sentence splitting.